native_client/args.h

```diff
@@ -39,39 +37,36 @@ int stream_size = 0;
 void PrintHelp(const char* bin)
 {
     std::cout <<
-    "Usage: " << bin << " --model MODEL [--lm LM --trie TRIE] --audio AUDIO [-t] [-e]\n"
+    "Usage: " << bin << " --model MODEL [--scorer SCORER] --audio AUDIO [-t] [-e]\n"
     "\n"
     "Running DeepSpeech inference.\n"
     "\n"
-    "	--model MODEL		Path to the model (protocol buffer binary file)\n"
-    "	--lm LM			Path to the language model binary file\n"
-    "	--trie TRIE		Path to the language model trie file created with native_client/generate_trie\n"
-    "	--audio AUDIO		Path to the audio file to run (WAV format)\n"
-    "	--beam_width BEAM_WIDTH	Value for decoder beam width (int)\n"
-    "	--lm_alpha LM_ALPHA	Value for language model alpha param (float)\n"
-    "	--lm_beta LM_BETA	Value for language model beta param (float)\n"
-    "	-t			Run in benchmark mode, output mfcc & inference time\n"
-    "	--extended		Output string from extended metadata\n"
-    "	--json			Extended output, shows word timings as JSON\n"
-    "	--stream size		Run in stream mode, output intermediate results\n"
-    "	--help			Show help\n"
-    "	--version		Print version and exits\n";
+    "\t--model MODEL\t\tPath to the model (protocol buffer binary file)\n"
+    "\t--scorer SCORER\t\tPath to the external scorer file\n"
+    "\t--audio AUDIO\t\tPath to the audio file to run (WAV format)\n"
+    "\t--beam_width BEAM_WIDTH\tValue for decoder beam width (int)\n"
+    "\t--lm_alpha LM_ALPHA\tValue for language model alpha param (float)\n"
+    "\t--lm_beta LM_BETA\tValue for language model beta param (float)\n"
+    "\t-t\t\t\tRun in benchmark mode, output mfcc & inference time\n"
+    "\t--extended\t\tOutput string from extended metadata\n"
+    "\t--json\t\t\tExtended output, shows word timings as JSON\n"
+    "\t--stream size\t\tRun in stream mode, output intermediate results\n"
+    "\t--help\t\t\tShow help\n"
+    "\t--version\t\tPrint version and exits\n";
     DS_PrintVersions();
     exit(1);
 }
 
 bool ProcessArgs(int argc, char** argv)
 {
-    const char* const short_opts = "m:a:l:r:w:c:d:b:tehv";
+    const char* const short_opts = "m:a:s:r:w:c:d:b:tehv";
     const option long_opts[] = {
             {"model", required_argument, nullptr, 'm'},
-            {"lm", required_argument, nullptr, 'l'},
-            {"trie", required_argument, nullptr, 'r'},
+            {"scorer", required_argument, nullptr, 'l'},
```

Comment:
Should the short argument be `'s'` and not `'l'`, to match `short_opts` ?

Reply:
`'s'` is used by `--stream`. It looks like there's more things mismatched here, I'll check the entire list.

Reply:
I went over the list and double checked things, and also made the order consistent across the three times these letters appear, so it's easier to spot mistakes.